test/cmdlineargs.jl

```diff
@@ -139,7 +139,7 @@ let exename = `$(Base.julia_cmd()) --startup-file=no --color=no`
     # handling of @projectname in --project and JULIA_PROJECT
     let expanded = abspath(Base.load_path_expand("@foo"))
         @test expanded == readchomp(`$exename --project='@foo' -e 'println(Base.active_project())'`)
-        @test expanded == readchomp(addenv(`$exename -e 'println(Base.active_project())'`, "JULIA_PROJECT" => "@foo", "HOME" => homedir()))
+        @test expanded == readchomp(addenv(`$exename -e 'println(Base.active_project())'`, "JULIA_PROJECT" => "@foo", "HOME" => homedir(), "JULIA_DEPOT_PATH" => ENV["JULIA_DEPOT_PATH"]))
```

Comment:
```suggestion
        let
            env = Dict{String, String}()
            env["JULIA_PROJECT"] = "@foo"
            env["HOME"] = homedir()
            if haskey(ENV, "JULIA_DEPOT_PATH")
                env["JULIA_DEPOT_PATH"] = ENV["JULIA_DEPOT_PATH"]
            end
            @test expanded == readchomp(addenv(`$exename -e 'println(Base.active_project())'`, env))
        end
```

Reply:
And similarly for any other tests that need this fix.

Reply:
Hm. I see what you did there. How about a shortcut? `Base.DEPOT_PATH[1]` will always yield the set `DEPOT_PATH` or will fall back on the default `~/.julia`

Reply:
That should work.

Reply:
I will commit your suggestion, but let's consider a shortcut.

Reply:
Oops. Let me fix the mess I created :D